shared-scripts.sh

```diff
@@ -465,3 +465,14 @@ function _emitCreateApiCanaryMetric {
     --dimensions branch=main,region=$CLI_REGION \
     --region us-west-2
 }
+
+function _emitCDKConstructCanaryMetric {
+  aws cloudwatch \
+    put-metric-data \
+    --metric-name $CANARY_METRIC_NAME \
+    --namespace amplify-graphql-api-construct-tests \
+    --unit Count \
+    --value $CODEBUILD_BUILD_SUCCEEDING \
+    --dimensions branch=release,region=$CLI_REGION \
```

Comment:
I notice the metric in the script is using `release` branch. Just a reminder to change the CR pointing the `main` once you finish the testing.

Reply:
Covered by [this task](https://app.asana.com/0/1206048977824382/1207039765008339/f)